### PR TITLE
fix: Remove cheqd-noded configure command from node [DEV-1902]

### PR DESCRIPTION
**Pull request by benyam7**
I removed the following:
 1.  `configure.go` file  and it's usage inside  `root.go `
 2.  `post_install` method(uses `configure` command) inside `installer.py` and it's usage as well.
 3.  `tests/e2e-complex/interactive-install/add-observer.sh` file
 4. `tests/e2e-complex/upgrade/cosmovisor/add-observer.sh`  file

@askolesov @anikitinDSR  Does this sound okay to you? Or is there anything that I am missing?

### CODE REVIEW

**Comment by ankurdotb** [comment]
Task linked: [DEV-1902 Remove cheqd-noded configure command from node & update documentation](https://app.clickup.com/t/6600954/DEV-1902)

File: installer/installer.py [hunk 1]
```diff
@@ -406,43 +406,13 @@ def install(self):
...
-        if self.interviewer.is_setup_needed:
-            self.post_install()
 
         if self.interviewer.init_from_snapshot:
             self.log("Downloading snapshot and extracting archive. This can take a *really* long time...")
             self.download_snapshot()
             self.untar_from_snapshot()
         self.print_success()
 
-    def post_install(self):
```
Comment (anikitinDSR) [inline installer/installer.py:418]: Let's keep this step, cause such action is still needed. It should be moved to `sed` command calling or something like this. I think, that there is another task for `installer.py`

**Review by anikitinDSR — COMMENTED** [review]
Please, keep `post_install` method just empty.

File: installer/installer.py (hunk 1 above)
Comment (benyam7) [reply installer/installer.py:418]: @anikitinDSR Okay,thanks! I kept it empty now.

Comment (askolesov) [reply installer/installer.py:418]: Yes, we can't just get rid of post install here. I would suggest to replace `configure` command with regex based replace

Comment (askolesov) [reply installer/installer.py:418]: @anikitinDSR What do you think about regexps?

**Comment by askolesov** [comment]
Also, I temporarily disabled proto breaking changes check, so we can see if build works or not.

**Review by askolesov — COMMENTED** [review]
Removal looks good, thanks! The only thing left here is to fix post install script. Take a look at what configure command did and try did the same with regex in python. Please ask me or @anikitinDSR  if you have any questions.

**Review by askolesov — APPROVED** [review]

**Pull request merged**